### ISSUE TITLE
Reindent menu don't work any more, CTRL-I does (1.2)

### ISSUE THREAD

**Issue by philibe** [rstudio/rstudio#4622]
### System details

```
RStudio Edition : RStudio Server (x86_64-pc-linux-gnu (64-bit) 
Running under: Ubuntu 16.04.4 LTS
RStudio Version : 1.2.1322
OS Version      : Windows 7 / Firefox 64.0.2
R Version       : R version 3.4.4 
```

<details><summary>Session_info</summary>

``` 
>  sessionInfo()
R version 3.4.4 (2018-03-15)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 16.04.4 LTS

Matrix products: default
BLAS: /usr/lib/libblas/libblas.so.3.6.0
LAPACK: /usr/lib/lapack/liblapack.so.3.6.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8     LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                  LC_ADDRESS=C               LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] formattable_0.2.0.1   knitr_1.22            rmarkdown_1.12        igraph_1.2.1          yaml_2.2.0            xts_0.11-1            zoo_1.8-4            
 [8] DiagrammeR_1.0.0      data.tree_0.7.5       bindrcpp_0.2.2        tidyr_0.8.2           stringr_1.3.1         gtools_3.5.0          devtools_1.13.6      
[15] glue_1.3.1            odbc_1.1.6            DBI_1.0.0             shinyBS_0.61          shinyjs_1.0           collapsibleTree_0.1.6 dplyr_0.7.8          
[22] statnet.common_4.0.0  xtable_1.8-3          shinydashboard_0.7.1  DT_0.4                shinyRGL_0.1.0        rgl_0.95.1441         shiny_1.2.0          

loaded via a namespace (and not attached):
 [1] viridis_0.5.1      bit64_0.9-7        jsonlite_1.6       viridisLite_0.3.0  assertthat_0.2.0   blob_1.1.1         pillar_1.2.2       lattice_0.20-35   
 [9] downloader_0.4     digest_0.6.17      RColorBrewer_1.1-2 promises_1.0.1     colorspace_1.3-2   htmltools_0.3.6    httpuv_1.4.5.1     plyr_1.8.4        
[17] XML_3.98-1.11      pkgconfig_2.0.2    purrr_0.2.4        scales_1.0.0       brew_1.0-6         later_0.7.5.9001   tibble_1.4.2       ggplot2_2.2.1     
[25] influenceR_0.1.0   withr_2.1.2        sourcetools_0.1.7  lazyeval_0.2.1     rgexf_0.15.3       crayon_1.3.4       magrittr_1.5       mime_0.5          
[33] evaluate_0.13      memoise_1.1.0      Rook_1.1-1         tools_3.4.4        hms_0.4.2          munsell_0.5.0      compiler_3.4.4     rlang_0.3.1       
[41] grid_3.4.4         rstudioapi_0.8     htmlwidgets_1.3    visNetwork_2.0.3   crosstalk_1.0.0    gtable_0.2.0       R6_2.3.0           gridExtra_2.3     
[49] bit_1.1-14         bindr_0.1.1        readr_1.1.1        stringi_1.2.2      Rcpp_0.12.19       dbplyr_1.2.1       xfun_0.6           tidyselect_0.2.5  
```

</details>

### Steps to reproduce the problem

Take the UI from https://shiny.rstudio.com/articles/basics.html and disturb indentation.

### Describe the problem in detail

Before : CTRL-A (select all lines)

Next:
- Menu / Code / Reindent Lines -> nothing happened
- CTRL-I -> the reindentation works.

### Describe the behavior you expected

Menu / Code / Reindent Lines ->  the reindentation will work again.

(Maybe there are other things unplugged  in the menu , I don't know.)

**Comment by pascaloettli** [comment]
It seems I can reproduce

### System details

    RStudio Edition : Server
    RStudio Version : 1.2.1335
    OS Version      : macOS Mojave 10.14.4 (client) / Ubuntu 18.04.2 (server)
    R Version       : 3.5.3
    Browser         : Firefox 66.0.3

**Comment by dfalty** [comment]
I'm able to reproduce this as well on Server Pro version 1.2.1335-1. Looks like this works fine on desktop versions. Thanks for the bug report!

**Comment by jmcphers** [comment]
@dfalty Are you able to reproduce on 1.1?

It looks like we don't execute Reindent unless the editor is focused, which is why it doesn't work from the menu.

https://github.com/rstudio/rstudio/blob/e8431c94a063a1c5cb089229eb078dcedc87b3e0/src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/AceEditorCommandDispatcher.java#L223-L225

**Comment by dfalty** [comment]
@jmcphers I just tried on the same server with 1.1.463 and 1.2.1335-2. It seems to work fine in 1.1 and doesn't in 1.2.

**Comment by kevinushey** [comment]
Should be fixed by https://github.com/rstudio/rstudio/pull/4675.

**Comment by ronblum** [comment]
Confirmed fixed in RStudio Server 1.2.1528 on Red Hat 7.6.
